### ISSUE TITLE
get_message_content error malformed JSON string

### ISSUE THREAD

**Issue by jezwhite** [line/line-bot-sdk-perl#108]
Hello,

I am trying to use get_message_content to fetch an image that has been sent from a user and I'm getting various error messages. The script below should show the behaviour

```
use LINE::Bot::API;
use warnings;
use strict;

print $LINE::Bot::API::VERSION."\n";

my $channel_secret = 'xyz';
my $channel_access_token = 'xyz';
my $messageID = 'xyz';

my $bot = LINE::Bot::API->new(
    channel_secret       => $channel_secret,
    channel_access_token => $channel_access_token,
);

my $res = $bot->get_message_content($messageID);
unless ($res->is_success) {
    # error handling
  my $errorStr = $res->message;
  warn "$errorStr\n";      
}
else {
  my $filename = $res->fh->filename;
  my $file;
  open my $fh, '<', $file or die "$!: $file";
}
```

Using valid channel_secret, channel_access_token and messageID I get this error:

```
1.14
Use of uninitialized value $res_content in concatenation (.) or string at /usr/local/share/perl/5.26.1/LINE/Bot/API/Client/Furl.pm line 167.
LINE Messaging API contents_download error: 404 https://api-data.line.me/message/11463169951287/content
        content= at LineTestGetImage.pl line 16.
malformed JSON string, neither tag, array, object, number, string or atom, at character offset 0 (before "(end of string)") at /usr/local/share/perl/5.26.1/LINE/Bot/API/Client/Furl.pm line 169.
```

There is scope for me to be using an incorrect message ID, I'm getting it from:

```
                 'message' => {
                                'contentProvider' => {
                                                       'type' => 'line'
                                                     },
                                'id' => '11463169951287',
                                'type' => 'image'
                              },
```

My environment details:

#37-Ubuntu SMP Mon Mar 18 16:15:14 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux
This is perl 5, version 26, subversion 1 (v5.26.1) built for x86_64-linux-gnu-thread-multi
SDK 1.14

**Comment by gugod** [comment]
@bedoshi can you take a look at this issue ?

**Comment by bedoshi** [comment]
noticed.
I will look this.

**Comment by bedoshi** [comment]
Sorry so much for my mistaking.
@gugod 
I opened PR, please check this when you have time.

**Comment by gugod** [comment]
Thanks @Bedoshi, we can release the fix in a few days.

/cc @jezwhite

**Comment by gugod** [comment]
Released in 1.16.
